src/clojure_tictactoe/game/game_rules.clj

```diff
@@ -0,0 +1,21 @@
+(ns clojure-tictactoe.game.game-rules
+  (:require [clojure-tictactoe.game.board :as board]))
+
+
+(defn assess-win 
+  [board]
+  (cond 
+    (= (board 0) (board 1) (board 2)) (board 0)
+    (= (board 3) (board 4) (board 5)) (board 3)
+    (= (board 6) (board 7) (board 8)) (board 6)
+    (= (board 0) (board 3) (board 6)) (board 0)
+    (= (board 1) (board 4) (board 7)) (board 1)
+    (= (board 2) (board 5) (board 8)) (board 2)
+    (= (board 0) (board 4) (board 8)) (board 0)
+    (= (board 2) (board 4) (board 6)) (board 2)))
```

Comment:
This conditionals can all be set to `let` variables to describe what they're each asserting. Right now, they're all sort of "magic" numbers/assertions.

Reply:
Yeah, this needs to be refactored anyway. Also I can't figure out how to make them all set to let vars.

Reply:
Never mind, figured it out. It still needs to be refactored but they're all set to lets now.